src/main/resources/application.properties

```diff
@@ -1,6 +1,9 @@
 # Kafka bootstrap applies to all topics
 kafka.bootstrap.servers=localhost:9092
 
+quarkus.http.port=8080
```

Comment:
I think these two should be 8085 as well. We can always override them in the production in any case with Openshift deployment settings.